Bookish/Bookish.csproj

```diff
@@ -14,4 +14,8 @@
       <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.6" />
     </ItemGroup>
 
+    <ItemGroup>

```

Comment:
I'm not entirely sure why this block is here?

Reply:
No idea - I got rid of it and everything still seems to work